### PR TITLE
Prepare upgrade

### PR DESCRIPTION
**Pull request by huangsuyu**

### CODE REVIEW

File: docs/smart-chain/guides/concepts/BEP86.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# BEP89
```
Comment (guagualvcha) [inline docs/smart-chain/guides/concepts/BEP86.md:1]: Bep89?

File: docs/smart-chain/guides/concepts/BEP86.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This BEP is designed to increase the rewards for relayer operators.
+
+## Extra Rewards From SystemReward
+
+This new BEP will introduce a new agovernable parameter `dynamicExtraIncentiveAmount` to the [RelayerIncentivize Contract](https://github.com/binance-chain/bsc-genesis-contract/blob/master/contracts/RelayerIncentivize.sol). These extra  amount of BNB which will be transferred from the `SystemReward` to the bsc relayer reward pool.
+
+## Claim Extra Rewards
+
+1. laim `dynamicExtraIncentiveAmount` from the SystemReward contract.
```
Comment (HaoyangLiu) [inline docs/smart-chain/guides/concepts/BEP86.md:15]: Claim

File: docs/smart-chain/developer/bind/mirror.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## What happen under the hood
+
+- Verify there is no pending mirror request
+- Check the total supply and token symbol is valid
+- Send a cross-chain package to issue a BEP2 token on Binance Chain
+- The newly created BEP2 token is locked until token holder send cross-chain transfer
+
+
+After binding, the liquid circulation is all on BSC.
```
Comment (HaoyangLiu) [inline docs/smart-chain/developer/bind/mirror.md:21]: `the liquid circulation is all on BSC.` -> `all liquid circulation is on BSC.`

File: docs/smart-chain/guides/concepts/BEP86.md (hunk 1 above)
Comment (huangsuyu) [reply docs/smart-chain/guides/concepts/BEP86.md]: Fixed1

File: docs/smart-chain/guides/concepts/BEP86.md (hunk 2 above)
Comment (huangsuyu) [reply docs/smart-chain/guides/concepts/BEP86.md]: Fixed

File: docs/smart-chain/developer/bind/mirror.md (hunk 3 above)
Comment (huangsuyu) [reply docs/smart-chain/developer/bind/mirror.md]: Updated

**Review by HaoyangLiu — APPROVED** [review]

**Pull request merged**